include/af/defines.h

```diff
@@ -316,6 +321,12 @@ typedef enum {
     AF_FIF_RAW          = 34    ///< FreeImage Enum for RAW Camera Image File
 } af_image_format;
 
+typedef enum {
+    AF_BACKEND_CPU,
+    AF_BACKEND_CUDA,
+    AF_BACKEND_OPENCL
```

Comment:
That's the behavior anyway right except that there is no specific enum for that.